src/layouts/components/Footer.tsx

```diff
@@ -1,314 +1,260 @@
-  const { t } = useTranslation("layout");
+export default function Footer(): JSX.Element {
+  const router = useRouter();
 
+  // default set language to be english to be directed to international telegram link
+  const [language, setLanguage] = useState(router.locale ?? "");
+
+  // eslint-disable-next-line consistent-return
+  useEffect(() => {
+    switch (router.locale) {
+      case "de":
+        return setLanguage("_DE");
+      case "fr":
+        return setLanguage("_FR");
+      default:
+        setLanguage("");
+    }
+  }, [router]);
   return (
-    <footer className="mt-20 mb-12">
-      <Container>
-        <div className="flex flex-col">
-          <Link
-            href={{ pathname: "/" }}
-            passHref
-            className="flex items-center cursor-pointer hover:text-primary-500"
-            data-testid="Footer.Logo"
-          >
-            <DeFiChainLogo className="w-40 lg:w-52 mb-2" />
-          </Link>
-          <h2
-            className="text-xl lg:text-2xl font-semibold"
-            data-testid="Footer.Heading"
-          >
-            {t("footer.title")}
-          </h2>
-        </div>
-        <div className="mt-10 flex flex-wrap space-y-8 xl:space-y-0">
-          <div className="flex flex-wrap max-w-sm lg:max-w-max w-full lg:w-2/3 xl:w-1/2 space-y-8 lg:space-y-0">
-            <div className="lg:w-1/2">
-              <FooterSectionSitemap />
-            </div>
-            <div className="lg:w-1/2">
-              <FooterSectionSocial />
-            </div>
+    <footer className="relative bg-dark-00">
+      <Container className="relative z-10 py-10 px-0 md:px-10 lg:px-0">
+        <div className="py-10 h-full w-full footer-background rounded-[30px] px-6 md:px-14 lg:px-24">
+          <div className="flex-col md:hidden">
+            <Link
+              href={{ pathname: "/" }}
+              passHref
+              className="flex items-center cursor-pointer hover:accent-dfc-gradient-text"
+              data-testid="Footer.Logo"
+            >
+              <DeFiChainLogo className="w-40 lg:w-52" />
+            </Link>
+            <SocialsRow
+              customStyle="text-dark-500 gap-4 flex flex-row pt-6 md:hidden"
+              language={language}
+            />
           </div>
-          <div className="flex flex-wrap w-full lg:w-1/2 xl:w-1/3 mx-auto lg:mx-0 xl:ml-auto space-y-6">
-            <div className="w-full">
-              <Script src="https://files.coinmarketcap.com/static/widget/currency.js" />
-              <div
-                className="coinmarketcap-currency-widget"
-                data-currencyid="5804"
-                data-base="USD"
-                data-ticker="true"
-                data-rank="true"
-                data-marketcap="true"
-                data-volume="true"
-                data-statsticker="false"
+          <div className="text-sm md:grid md:grid-cols-2 md:grid-flow-row md:gap-[62px] lg:flex lg:flex-row md:justify-between md:pb-[86px]">
```

Comment:
column gap for 1440px screen (lg) should be `56px`

Reply:
column gap for 768px screen (md) should be `76px`

Reply:
Removed props from Container component to handle this